styles/index.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
```

Comment:
CSS file should contain the style and layout of the website, It is a good practice. It should be having selector and properties of the html elements rather than the link of the website. [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). for more information regarding this :blush: